packages/expo-location/ios/Providers/BaseStreamer.swift

```diff
@@ -0,0 +1,14 @@
+// Copyright 2024-present 650 Industries. All rights reserved.
+
+import CoreLocation
+import ExpoModulesCore
+
+protocol Streamer {
+  func stopStreaming()
+}
+
+internal class BaseStreamer: BaseLocationProvider, Streamer {
+  func stopStreaming() {
+    // Default empty implementation
+  }
+}
```

Comment:
```suggestion
internal class BaseStreamer: BaseLocationProvider {
  func stopStreaming() {
    // Default empty implementation
  }
}
```
Can we merge it? You're only using the `BaseStreamer` anyway.

Reply:
Good point as always @lukmccall - thanks!!